### PR TITLE
Update 1

### PR DESCRIPTION
**Pull request by ManasiGandhi**

### CODE REVIEW

File: ref/general/microshed.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+== Why integration testing
+
+In integration testing, you can put together individual pre-tested modules and test them as a group. Integration testing follows unit testing. The modules used are previously unit tested, but now they are now tested as a whole to evaluate compliance and functionality.
+
+
+== What challenges does Microshed resolve
+
+An application might run differently in production as compared to test/dev environment. This can cuse a test to fail in production though it passed in  dev/test. You can address this issue by testing your application in the same container that you’ll use in production.
+
+Running tests in development mode is convenient for local development, but it can be tedious to test against a running Open Liberty server in non-development scenarios such as CI/CD pipelines. For this reason, MicroShed Testing can start and stop the application runtime before and after the tests are run. This process is primarily accomplished by using Docker and Testcontainers.
```
Comment (Charlotte-Holt) [inline ref/general/microshed.adoc]: Are you using this just as a placeholder?? If so, no problem, but I think this was taken directly from the guide and we shouldn't duplicate content exactly.

File: ref/general/microshed.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+
+== How it works (with code snippets)
+
+
+
+You can use MicroShed Testing to develop integration tests for your Open Liberty application. You test your application from outside its container so that the tests run against the same image that you use in production.
+
+To write a test with MircoShed testing, first add microshed-testing-testcontainers and junit-jupiter artifact IDs as test-scoped dependencies, as shown in the following example:
```
Comment (Charlotte-Holt) [inline ref/general/microshed.adoc]: Suggestion for this sentence:
To write a test with MircoShed testing, first add `microshed-testing-testcontainers` and `junit-jupiter` artifact IDs as test-scoped dependencies, as shown in the following example:

File: ref/general/microshed.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<!-- Any compatible version of JUnit Jupiter 5.X will work -->
+<dependency>
+    <groupId>org.junit.jupiter</groupId>
+    <artifactId>junit-jupiter</artifactId>
+    <version>5.4.2</version>
+    <scope>test</scope>
+</dependency>
+----
+
+Next, create a test class with the `MicroShedTest` annotation. Then, create a public `static MicroProfileApplication` method and inject one or more `public static JAX-RS` resources, as shown in the following example:
```
Comment (Charlotte-Holt) [inline ref/general/microshed.adoc]: Suggestion for this paragraph:
Next, create a test class with the `@MicroShedTest` annotation. Then, create a public static `MicroProfileApplication` method and inject one or more public static JAX-RS resources, as shown in the following example:

Although `MicroProfileApplication` might be an object in this example? Not sure on that.

File: ref/general/microshed.adoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+[source,java]
+----
+    FROM open-liberty:microProfile3
+ADD build/libs/$APP_FILE /config/dropins
+COPY src/main/liberty/config /config
+----
+
+
+Ready to try how to use MicroShed Testing to test a MicroProfile or Jakarta EE application? Check out our guide https://openliberty.io/guides/microshed-testing.html[Testing a MicroProfile or Jakarta EE application]
```
Comment (Charlotte-Holt) [inline ref/general/microshed.adoc]: Suggestion for this paragraph:
Ready to try using MicroShed Testing to test a MicroProfile or Jakarta EE application? Check out the guide https://openliberty.io/guides/microshed-testing.html[Testing a MicroProfile or Jakarta EE application].

File: ref/general/microshed.adoc [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    public static MicroProfileApplication app = new MicroProfileApplication()
+                    .withAppContextRoot("/myservice");
+
+    @Inject
+    public static MyService mySvc;
+
+    // write @Test methods as normal
+----
+
+    In cases where a Dockerfile or container image is not available, use the microshed-testing-liberty adapter. This adapter produces a container image that is similar to the following Dockerfile:
```
Comment (Charlotte-Holt) [inline ref/general/microshed.adoc]: Should microshed-testing-liberty adapter be `microshed-testing-liberty` adapter?

File: ref/general/microshed.adoc (hunk 1 above)
Comment (ManasiGandhi) [reply ref/general/microshed.adoc]: Yes I am using it as a placeholder for now.

File: ref/general/microshed.adoc (hunk 3 above)
Comment (ManasiGandhi) [reply ref/general/microshed.adoc]: Again a place holder for now. Will discuss it with the SME and update.

File: ref/general/microshed.adoc (hunk 2 above)
Comment (ManasiGandhi) [reply ref/general/microshed.adoc]: The suggestion is the same as the content in the topic. Can you please point to the chnages you are recommending.

**Pull request merged**